### ISSUE TITLE
Screen Reader cannot read names of some elements in 'Contact support' screen

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7631]
The following Accessibility issues are flagged by Accessibility Insights in the ‘Contact support’ screen – see screenshot. The screen reader will not be able read names if they are missing

!image-20230729-001631.png|width=1523,height=818!

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5293)
┆Reporter: Vinoo Cherian

**Comment by data-sync-user** [comment]
➤ Raluca Monica Ilban commented:

Note for QA: as mentioned in https://mozilla-hub.atlassian.net/browse/VPN-5783?focusedCommentId=799137 ( https://mozilla-hub.atlassian.net/browse/VPN-5783?focusedCommentId=799137|smart-link ) , we need to check if the subtitle on the home screen is narrated.

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

The Expand/Collapse pattern cannot be controlled by us because it is in the Qt framework, but the name issues have been fixed.
